app/gui/src/model/module/synchronized.rs

```diff
@@ -113,22 +113,19 @@ impl ParsedContentSummary {
     }
 }
 
-/// The information about state of the module currently held in LanguageServer.
+/// The information about module's state currently held in LanguageServer.
 #[derive(Clone, Debug)]
 enum LanguageServerContent {
     /// The content is synchronized with our module state after last fully handled notification.
     Synchronized(ParsedContentSummary),
-    /// The content is not synchronized with our module state after last fully handled
-    /// notification, probably due to connection error when sending update.
-    Desynchronized(ContentSummary),
+    /// The content is not known due to unrecognized error received from the Language Server after
+    /// applying last update. We don't know if, and in what extend was it applied.
+    Unknown,
 }
 
-impl LanguageServerContent {
-    fn summary(&self) -> &ContentSummary {
-        match self {
-            LanguageServerContent::Synchronized(content) => &content.summary,
-            LanguageServerContent::Desynchronized(content) => content,
-        }
+impl Default for LanguageServerContent {
+    fn default() -> Self {
+        Self::Unknown
     }
 }
```

Comment:
nit: it's possible to derive `Default` and mark the needed variant with the `#[default]` attribute.

Reply:
Indeed. For some reason, I thought it's possible only when _all_ variants have no fields.